xcdat/temporal.py

```diff
@@ -285,24 +292,29 @@ def group_average(
             predefined seasons are passed, configs for custom seasons are
             ignored and vice versa.
 
-            Configs for predefined seasons:
+            * "drop_incomplete_seasons" (bool, by default False)
```

Comment:
@tomvothecoder I am little bit concerned that this can be a breaking change if someone was using drop_incomplete_djf in their code and it starts not working when they updated xcdat version. I wonder this change can be incremental change, e.g., if drop_incomplete_djf is used, raise a will-be-deprecated warning message that encourages to use drop_incomplete_season instead. Internally, drop_incomplete_djf can be remapped to drop_incomplete_season until it is fully deprecated. Would this be a reasonable thing to consider?